RELEASE.md

```diff
@@ -1,47 +1,62 @@
-# Release Process
+# Release
 
-The following is the release process you should follow to publish a new version of `ember-a11y-testing`.
+Releases are mostly automated using
+[release-it](https://github.com/release-it/release-it/) and
+[lerna-changelog](https://github.com/lerna/lerna-changelog/).
 
-## Update The Changelog
 
-First, we need to update the `CHANGELOG.md` file for the project. This requires having [`git-extras`](https://github.com/tj/git-extras) installed.
+## Preparation
 
-Checkout the current `master` version of the repo and run:
+Since the majority of the actual release process is automated, the primary
+remaining task prior to releasing is confirming that all pull requests that
+have been merged since the last release have been labeled with the appropriate
+`lerna-changelog` labels and the titles have been updated to ensure they
+represent something that would make sense to our users. Some great information
+on why this is important can be found at
+[keepachangelog.com](https://keepachangelog.com/en/1.0.0/), but the overall
+guiding principle here is that changelogs are for humans, not machines.
 
-```bash
-git changelog
-```
-
-Clean up the generated changelog by inserting the appropriate verion number and removing merge commits and previous release commits if necessary.
+When reviewing merged PR's the labels to be used are:
 
-Review the changes and then commit them with a message like:
+* breaking - Used when the PR is considered a breaking change.
+* enhancement - Used when the PR adds a new feature or enhancement.
+* bug - Used when the PR fixes a bug included in a previous release.
+* documentation - Used when the PR adds or updates documentation.
+* internal - Used for internal changes that still require a mention in the
+  changelog/release notes.
```

Comment:
Just wondering, what would a minor/patch dependency update fall under, enhancement?

Reply:
@drewlee wouldn't that be considered internal?

Reply:
Yeah, that makes sense @MelSumner. Thanks for clarifying.